src/components/notifications/ExceptionMessage.tsx

```diff
@@ -0,0 +1,107 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import React, { useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import { copyTextToClipboard } from '../../helpers/copyHandler';
+import toastCopy from '../../assets/icons/toast-copy.svg';
+
+interface ExceptionMessageProps {
+    notificationType: "exceptionMessage";

```

Comment:
it seems these 3 components are very much the same with the same copy-pasted styles, can we make this markup and styles to be the base of these components? it would accept as props title and object to copy, 
```
const ExceptionMessage = () => <MessageBase title={} objectToCopy={}/>
```